test/fixtures/dialects/oracle/comment.yml

```diff
@@ -0,0 +1,19 @@
+# YML test files are auto-generated from SQL files and should not be edited by
+# hand. To help enforce this, the "hash" field in the file must match a hash
+# computed by SQLFluff when running the tests. Please run
+# `python test/generate_parse_fixture_yml.py`  to generate them after adding or
+# altering SQL files.
+_hash: 4498336da34808ac0fc2ede6d2dca06d03821ac28af9fc8825a3f7e6e4b80cf2
+file:
+  statement:
+    comment_statement:
+    - keyword: COMMENT
+    - keyword: 'ON'
+    - keyword: COLUMN
+    - column_reference:
+      - identifier: employees
+      - dot: .
+      - identifier: job_id
+    - keyword: IS
+    - literal: "'abbreviated job title'"
+  statement_terminator: ;
```

Comment:
@r0fls Looks like you need to re-run `tox -e generate-fixture-yml` to add the include the extra unit tests in the fixture

Reply:
Thank you, still getting `RuntimeError: Grammar refers to 'IndextypeKeywordSegment' which was not found in the oracle dialect` which is why I had it commented out 🤔 I grep'd around for `ObjectReferenceKeyword` and `TableReferenceKeyword` but couldn't find any examples of adding this to work from.

Reply:
You need to ass `"INDEXTYPE"` to the keywords (in same way you added `"COMMENT"`,`"ON"`,`"UPDATE"`

Reply:
Whoops, thank you! Added ✅